src/main/java/no/nav/veilarbaktivitet/arena/ArenaForhaandsorienteringDAO.java

```diff
@@ -0,0 +1,73 @@
+package no.nav.veilarbaktivitet.arena;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.veilarbaktivitet.avtaltMedNav.Forhaandsorientering;
+import no.nav.veilarbaktivitet.db.Database;
+import no.nav.veilarbaktivitet.domain.Person;
+import no.nav.veilarbaktivitet.domain.arena.ArenaAktivitetDTO;
+import no.nav.veilarbaktivitet.util.EnumUtils;
+import org.springframework.stereotype.Repository;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Repository
+@RequiredArgsConstructor
+public class ArenaForhaandsorienteringDAO {
+
+    private final Database database;
+
+    void insertForhaandsorientering(String arenaaktivitetId, Person.AktorId aktorId, Forhaandsorientering forhaandsorientering) {
+        String forhaandsorienteringType = null;
+        String forhaandsorienteringTekst = null;
+        Date fhoOpprettetDato = new Date();
+
+        if(forhaandsorientering != null) {
+            forhaandsorienteringType = forhaandsorientering.getType().name();
+            forhaandsorienteringTekst = forhaandsorientering.getTekst();
+        }
+
+        //language=SQL
+        database.update("INSERT INTO ARENA_FORHAANDSORIENTERING(arenaaktivitet_id, aktor_id, fho_type, fho_tekst, fho_opprettet_dato) " +
+                        "VALUES (?,?,?,?,?)",
+                arenaaktivitetId,
+                aktorId.get(),
+                forhaandsorienteringType,
+                forhaandsorienteringTekst,
+                fhoOpprettetDato
+        );
+
+        log.info("opprettet {}", forhaandsorientering);
+    }
+
+    List<ArenaForhaandsorienteringData> hentForhaandsorienteringer(List<ArenaAktivitetDTO> aktiviteter) {
+        if(aktiviteter.isEmpty()) {
```

Comment:
Trenger ikke denne egentlig, får samme resultat uten.

Reply:
Du får ugyldig sql i h2

Reply:
Kanskje skrive en kommentar i koden om det da? Kan spare nestemann for litt arbeid

Reply:
Ser ut itl å vere det samme i de flesta andre databasemotorer.